CONTRIBUTING.md

```diff
@@ -33,6 +33,15 @@ This ensures that different users have the same versions of packages installed.
 See also [collaborating with renv](https://rstudio.github.io/renv/articles/collaborating.html).
 The first time you open the RStudio project `renv` should automatically download and install the appropriate version of `renv` into the project library.
 After this has completed, you can use `renv::restore()` to restore the project library locally on your machine.
+In case you need another package than the onces that are installed (see the [DESCRIPTION file](DESCRIPTION)) for this project, proceed as follows:
+
+- `renv::install("nameofthepackage")`
+- add the name of the package to the [DESCRIPTION file](DESCRIPTION)
+- `renv::snapshot()`: type 'y' in the console when asked to update [the lock file](renv.lock)
+- commit the changed [lock file](renv.lock)
```

Comment:
This procedure may cause headache for novices. Perhaps provide a single `protocolhelper` function that accepts a vector of package names and does this all at once?

@hansvancalster @ElsLommelen this should not delay this PR, but if considered a useful idea, be turned into an issue at `protocolhelper`.

Reply:
I have a related issue that I need to address with @hansvancalster: for now the renv packages are installed in the docker images (all three) itself, which has the advantage that the docker containers have a full installation already (which makes the GHA quick), but the disadvantage that we need to build a new docker image after adding new packages (which might be necessary anyway if a package installation needs additional software to run on Ubuntu).  (And also: is this installation of all renv packages necessary for all three GHA, i.e. in which steps do we need to render the protocols?)

In the current situation, we have to take action anyway (rebuild the docker image), so maybe in this case we can just add this package to renv as well?  Then they just have to ask us to add new packages, which has the advantage that we can also 'consider carefully' ourselves (see text on next lines).

And to be honest, regarding the many difficulties I experienced to have these 'renv' packages installed in docker, and considering the fact that also less experienced R users may add protocols, I think it may be a good idea anyway to save novices from a headache add these packages ourselves.  As these are protocols, I expect mainly text with some illustrations or tables, so I suppose the list of packages is rather limited and will hardly grow after the first dozen of packages are added?  Or am I mistaken and could it be a large effort for us?

Anyway, to be added either as an issue, either as a discussion topic in the PR I submit to add the automation...

Reply:
Sounds all very reasonable to me. I leave the further considerations to you and Hans.